### PR TITLE
ci: Add script checking for `#pragma once` to CI

### PR DESCRIPTION
**Pull request by benjaminhuth**
I noticed a missing `#pragma once` in a file in `Core`, and thought it would be nice to check this in the CI.

### CODE REVIEW

**Review by paulgessinger — APPROVED** [review]
Seems like a good idea to me!

**Review by andiwand — APPROVED** [review]
cool!

looks like we are missing some includes now which cascaded through before

File: CI/check_pragma_once.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+ec=0
+
+for file in $(find Core Examples Tests Plugins -name "*.hpp"); do
+    res=$(grep "#pragma once" $file)
```
Comment (gagnonlg) [inline CI/check_pragma_once.sh:6]: maybe pedantic (feel free to ignore) but you make this more robust, e.g. `grep -e '^[[:space:]]*#pragma once' test.cxx` . This way you will catch instance where the pragma is commented-out for instance

Comment (benjaminhuth) [reply CI/check_pragma_once.sh]: nice, I changed the `grep` command!

**Review by andiwand — APPROVED** [review]

**Pull request merged**